src/StackExchange.Redis/RedisServer.cs

```diff
@@ -856,14 +866,22 @@ public Task SentinelFailoverAsync(string serviceName, CommandFlags flags = Comma
             return ExecuteAsync(msg, ResultProcessor.SentinelArrayOfArrays);
         }
 
-        public KeyValuePair<string, string>[][] SentinelSlaves(string serviceName, CommandFlags flags = CommandFlags.None)
+        KeyValuePair<string, string>[][] IServer.SentinelSlaves(string serviceName, CommandFlags flags)
+            => SentinelReplicas(serviceName, flags);
+
+        public KeyValuePair<string, string>[][] SentinelReplicas(string serviceName, CommandFlags flags = CommandFlags.None)
         {
+            // note: sentinel does not have "replicas" terminology at the current time
             var msg = Message.Create(-1, flags, RedisCommand.SENTINEL, RedisLiterals.SLAVES, (RedisValue)serviceName);
             return ExecuteSync(msg, ResultProcessor.SentinelArrayOfArrays);
         }
 
-        public Task<KeyValuePair<string, string>[][]> SentinelSlavesAsync(string serviceName, CommandFlags flags = CommandFlags.None)
+        Task<KeyValuePair<string, string>[][]> IServer.SentinelSlavesAsync(string serviceName, CommandFlags flags)
```

Comment:
API break? It's a recent addition, but still

Reply:
Thought I forwarded. Should have.

Reply:
Ah yes this one's internal :+1: All good